cuebot/src/test/resources/opencue.properties

```diff
@@ -3,6 +3,7 @@ spring.velocity.checkTemplateLocation=false
 
 grpc.cue_port=8453
 grpc.rqd_server_port=${CUEBOT_GRPC_RQD_SERVER_PORT:50051}
+grpc.use_fqdn=true
```

Comment:
Ideally we could run some tests with `false`, and some with `true`, to test both codepaths. Could we set the default here to `false` and modify the setting for some tests?

Having the default in `main` be `false` is good to avoid affecting existing users, but it means we won't be testing the default codepath, even though the differences are minor at this point.

Reply:
This makes sense but I'm not sure of the best way to approach it.  For now I've set the default back to false in the test properties so at least the default codepath is tested.  Is there an example elsewhere of overriding properties in the tests?